Weather/ViewModel/TodayForecastViewModel.swift

```diff
@@ -61,4 +51,22 @@ class TodayForecastViewModel: ViewModel {
 	func accept(intent: Intent) {
 		intents.accept(intent)
 	}
+  
+  private func byEvents(_ event: Event) -> Intent {
+    if event is LoadTodayForecastEvent {
```

Comment:
Abi bunun object oriented yaklaşım olduğunu okumuştum. Ne zaman *is* yazacağın yer olursa onu generic ayrı bir func yapmayı öneriyordu

Reply:
Ya bende ilk seferde öyle yazmaya çalıştım ama bu extension da çok can sıktı